src/Infrastructure/Databases/EFUserRepository.cs

```diff
@@ -87,7 +88,11 @@ public void EnsureAdminCreated()
 
     private void CreateAdmin()
     {
-        var admin = new User { Name = AdminUserName, Company = "Prodigo", LatestPaymentDate = DateTime.MaxValue, };
+        string? adminPasswordSecret = Environment.GetEnvironmentVariable(ConfigurationKeys.AdminPassword);
+        Console.WriteLine(adminPasswordSecret);
+        adminPasswordSecret = BCrypt.Net.BCrypt.HashPassword(adminPasswordSecret);
+        Console.WriteLine(adminPasswordSecret);
```

Comment:
Debugging artifacts 😩 
```suggestion
        adminPasswordSecret = BCrypt.Net.BCrypt.HashPassword(adminPasswordSecret);
```